server/views/pages/optOut/optOut.njk

```diff
@@ -16,21 +15,25 @@
       },
       id: "otherReason",
       name: "otherReason",
+      value: submittedForm.otherReason,
       classes: "govuk-input--width-30",
       errorMessage: validationErrors | findError('otherReason')
     }) }}
 {% endset %}
 
 {% block content %}
     <div class="govuk-width-container">
+      <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
+            {% include 'partials/errorSummary.njk' %}
```

Comment:
extracted out an include to display error summaries if there are any validation errors